source/platforms/kubernetes.md

```diff
@@ -46,6 +46,213 @@ spec:
           name: notebook
 ```
 
+````{dropdown} Optional: Extended notebook configuration to enable launching multi-node Dask clusters
+:color: info
+:icon: info
+
+Deploying an interactive single-user notebook can provide a great place to launch further resources. For example you could install `dask-kubernetes` and use the [dask-operator](../tools/kubernetes/dask-operator) to create multi-node Dask clusters from your notebooks.
+
+To do this you'll need to create a couple of extra resources when launching your notebook `Pod`.
+
+### RBAC Service account and role
```

Comment:
Maybe for myself, what does `RBAC` mean here?

Reply:
Thanks for flagging this acronym as being defined without a definition. It stands for Role Based Acces Control and is the way Kubernetes manages authentication. https://kubernetes.io/docs/reference/access-authn-authz/rbac/

I think it is probably unnecessary in the title anyway so my suggestion would be to just remove it rather than define it.

```suggestion
### Service account and role
```